compiler-core/src/javascript.rs

```diff
@@ -41,13 +42,16 @@ pub struct Generator<'a> {
     module_scope: im::HashMap<EcoString, usize>,
     current_module_name_segments_count: usize,
     target_support: TargetSupport,
+
+    typescript: TypeScriptDeclarations,
```

Comment:
```suggestion
    typescript: TypeScriptDeclarations,
```
remove the extra newline